gusb/meson.build

```diff
@@ -64,6 +64,7 @@ gusb = library(
     'gusb-source.c',
     'gusb-util.c',
     'gusb-version.c',
+    'gusb-json-helper.c',
```

Comment:
Tiny nit, maybe gusb-json-common.c as a prefix. Helper is usually for userdata callbacks. Then lgtm